freedom-mee_header-generator.c++

```diff
@@ -174,9 +183,25 @@ static void write_config_file(const fdt &dtb, fstream &os, std::string cfg_file)
     os << "                    &__mee_dt_" << n.handle() << ",\n";
   };
 
+  /*
+   * Dont have a better way of using macros generated here in structure defintion.
+   * A chicken and egg issue, so for now, simply use compile flag to limit the scope
+   * for inclusion.
+  */
+  os << "#ifdef __MEE_MACHINE_MACROS\n";
```

Comment:
This adds the ifdef block even if we don't have any matches for "sifive,clic0". Do we care about having a stray

```
#ifdef __MEE_MACHINE_MACROS
#endif
```

here or there in the machine header?

Reply:
This is intended, thus the ifdef check is a generic tag. Although only clic the need this part now,
In future, other tags may be added here.